sphinx-4-4-release-other-ecosystem-news.rst

```diff
@@ -0,0 +1,133 @@
+      from the Sphinx ecosystem of extensions and themes.
+
+Sphinx 4.4 release and other ecosystem news
+===========================================
+
+In this post we spread the word about
+the most relevant news of the Sphinx ecosystem of the past weeks,
+including Sphinx itself as well as extensions and themes developed by the community.
+
+Sphinx 4.4 release
+------------------
+
+**Sphinx 4.4** was released on January 17th with numerous changes, including:
+
+Better control of intersphinx references with the ``:external:`` role
+   One of the power features of Sphinx is the ability to include
+   cross references across projects, using :std:doc:`intersphinx <sphinx:usage/extensions/intersphinx>`.
+   By default, Sphinx transparently attempts to locate objects in external projects
+   if they are not found in the current one,
+   which is useful but also can become confusing in some cases.
+   The new :rst:role:`external` role, coupled with the
+   `intersphinx_disabled_reftypes <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_disabled_reftypes>`_
+   configuration introduced in Sphinx 4.3,
+   enables documentation writers to control
+   when intersphinx references should be used.
+
+   You can use the new ``:external:`` role as follows:
+
+   .. code-block:: rst
+
+      External reference: :external:py:class:`zipfile.ZipFile`.
+      External reference with constrained domain lookup: :external+python:py:class:`zipfile.ZipFile`.
+
+   See more information in :rst:role:`the official documentation <external>`.
+
+New asynchronous methods to load JavaScript files
+   The method :py:meth:`sphinx:sphinx.application.Sphinx.add_js_file`
+   now accepts a new ``loading_method`` parameter that can either be ``"async"`` or ``"defer"``,
+   which results in the script being loaded with the
+   `async <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async>`_ or
+   `defer <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer>`_
+   HTML attributes respectively.
+
+Proper error messages when autosummary does not find a package
+   Sphinx ships the :std:doc:`autosummary <sphinx:usage/extensions/autosummary>` extension
+   to automatically generate API pages of a Python library.
+   However, autosummary used to raise misleading error messages if the target library failed to import,
+   which resulted in user frustration.
+   This problem has now been fixed and autosummary properly points out the reason of the ``ImportError``.
+
+You can read the `complete Sphinx 4.4.0 release
+notes <https://www.sphinx-doc.org/en/master/changes.html#release-4-4-0-released-jan-17-2022>`_ online.
+
+Using Sphinx 4.4 on Read the Docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sphinx 4.4 is supported on Read the Docs. By default, RTD will install the latest version for you,
+as described in :ref:`our documentation <readthedocs:builds:external dependencies>`.
```

Comment:
I'm happy this is very clear now in our documentation, it's easy to explain to user ("we install the latest version available"), and there are no surprises :100:. Way better than pinning to an old version and having to explain the "why" to users.